packages/matchbox/src/components/Expandable/Expandable.js

```diff
@@ -1,9 +1,76 @@
 import React from 'react';
 import Proptypes from 'prop-types';
-import classnames from 'classnames';
+import styled from 'styled-components';
+import { margin } from 'styled-system';
+import { createPropTypes } from '@styled-system/prop-types';
 import { KeyboardArrowRight } from '@sparkpost/matchbox-icons';
 import { onKeys } from '../../helpers/keyEvents';
-import styles from './Expandable.module.scss';
+import { expandable, header, title, subtitle, arrow, contentWrapper } from './styles';
+
+import Accent from './Accent';
+import { Box } from '../Box';
+
+const StyledExpandable = styled('div')`
+  ${expandable}
+  ${margin}
+`;
+
+export const StyledHeader = styled(Box)`
+  ${header}
+`;
+
+export const StyledArrow = styled('div')`
+  ${arrow}
+`;
+
+export const StyledTitle = styled('h3')`
+  ${title}
+`;
+
+const StyledSubtitle = styled('h6')`
+  ${subtitle}
+`;
+
+const StyledContentWrapper = styled('div')`
+  ${contentWrapper}
+`;
+
+function Icon({ icon }) {
+  return icon ? (
+    <Box flex="0" minWidth="40px" maxWidth="40px" mr="500">
+      {icon}
+    </Box>
+  ) : null;
+}
+
+function Title({ title }) {
+  return <StyledTitle>{title}</StyledTitle>;
+}
+
+function Subtitle({ subtitle }) {
+  return <StyledSubtitle>{subtitle}</StyledSubtitle>;
+}
```

Comment:
I'm not sure about the advantage of defining new components in this way? It may be less repetitive to just use `<StyledTitle>` directly?